web/src/beta/lib/core/engines/Cesium/Feature/Tileset/hooks.ts

```diff
@@ -516,11 +529,34 @@ export const useHooks = ({
       : null;
   }, [isVisible, tileset, url, type, meta]);
 
+  const imageBasedLighting = useMemo(() => {
```

Comment:
I think we should support this for model as well:pray:
Then we can test the environment map more easily by using Cesium sandcastle's glb file.
https://sandcastle.cesium.com/index.html?src=Image-Based%20Lighting.html

Reply:
Thank you for helping implement the IBL for model!